crates/kubelet/src/server.rs

```diff
@@ -20,7 +26,14 @@ pub async fn start_webserver<T: 'static + Provider + Send + Sync>(
     provider: Arc<Mutex<T>>,
     address: &SocketAddr,
 ) -> Result<(), failure::Error> {
-    let service = make_service_fn(move |_conn: &AddrStream| {
+    let identity = tokio::fs::read("/home/rylevick/krustlet/certificate.pfx")
```

Comment:
Obviously this is not correct 😛 - The question is how should this be configured.

Reply:
Definitely pull it from the `Config` object and expose it as a flag and env variable. As for local dev, we should probably have a `just` target that makes a self-signed cert and puts it in a default location